homeassistant/components/raspberry_pi/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "raspberry_pi",
+  "name": "Raspberry Pi",
+  "config_flow": false,
+  "documentation": "https://www.home-assistant.io/integrations/raspberry_pi",
+  "requirements": [],
```

Comment:
Empty items.

Reply:
We add them to make people aware they exist. It's easier to remove something than to read the docs and realize something is possible.

Reply:
OK, but if we don't want them, we could have empty items fail CI?

Reply:
linting them out would be good 👍